stubs/tensorflow/tensorflow/keras/initializers.pyi

```diff
@@ -0,0 +1,48 @@
+from _typeshed import Incomplete
+from collections.abc import Callable
+from typing import Any, overload
+from typing_extensions import Self, TypeAlias
+
+from tensorflow import Tensor, _DTypeLike, _ShapeLike, _TensorCompatible
+
+class Initializer:
+    def __call__(self, shape: _ShapeLike, dtype: _DTypeLike | None = None) -> Tensor: ...
+    def get_config(self) -> dict[str, Any]: ...
+    @classmethod
+    def from_config(cls, config: dict[str, Any]) -> Self: ...
+
+class Constant(Initializer):
+    def __init__(self, value: _TensorCompatible = 0) -> None: ...
+
+class GlorotNormal(Initializer):
+    def __init__(self, seed: int | None = None) -> None: ...
+
+class GlorotUniform(Initializer):
+    def __init__(self, seed: int | None = None) -> None: ...
+
+class TruncatedNormal(Initializer):
+    def __init__(self, mean: _TensorCompatible = 0.0, stddev: _TensorCompatible = 0.05, seed: int | None = None) -> None: ...
+
+class RandomNormal(Initializer):
+    def __init__(self, mean: _TensorCompatible = 0.0, stddev: _TensorCompatible = 0.05, seed: int | None = None) -> None: ...
+
+class RandomUniform(Initializer):
+    def __init__(self, minval: _TensorCompatible = -0.05, maxval: _TensorCompatible = 0.05, seed: int | None = None) -> None: ...
+
+class Zeros(Initializer): ...
+
+constant = Constant
+glorot_normal = GlorotNormal
+glorot_uniform = GlorotUniform
+truncated_normal = TruncatedNormal
+zeros = Zeros
+
+_Initializer: TypeAlias = str | Initializer | Callable[[_ShapeLike], Tensor] | dict[str, Any] | None
+
+@overload
+def get(identifier: None) -> None: ...
+@overload
+def get(identifier: str | Initializer | dict[str, Any]) -> Initializer: ...
+@overload
+def get(identifier: _Initializer) -> Initializer | None: ...
```

Comment:
Same as before, seems like this is only the `Callable`.

Reply:
Fixed. For this one re-reading [source](https://github.com/keras-team/keras/blob/v2.11.0/keras/initializers/__init__.py#L153-L201) I missed one case (type[Initializer] is also allowed).